x/marker/spec/03_messages.md

```diff
@@ -243,6 +244,17 @@ This service message is expected to fail if:
   - The given administrator address does not currently have the "transfer" access granted on the marker
   - The marker types is not `RESTRICTED_COIN`
 
+## Msg/IbcTransferRequest
+
+Ibc transfer Request defines the Msg/IbcTransfer request type.  A ibc transfer request is used to transfer coin between two
```

Comment:
A few typos. Feel free to use the following or take pieces from it...

The IbcTransferRequest is used to transfer `RESTRICTED_COIN` type markers to another chain via ibc. These coins have their `send_enabled` flag disabled by the `bank` module and thus cannot be sent using a normal `send_coin` operation.  

### Note
A transfer request also requires a signature from an account with the transfer permission as well as approval from the account the funds will be withdrawn from.

Reply:
Updated.